docs/best-practices/tech-debt.md

```diff
@@ -0,0 +1,218 @@
+Ops and Permission flags are expected to be long-lived and remain in code.
+
+## Lifecycle of a Feature
+Features that are not expected to stay in code forever (ie. Release and Experiment Features) generally go through a
+similar set of lifecycle steps.
+
+For a release feature:
+1. Feature Variables are added to code and the Feature is set up in DevCycle
+2. Feature is enabled for particular groups of users throughout development
+3. Feature is enabled for all users when it is complete
+4. Feature is considered "complete" once it is fully released and stable, and the ability to turn it off is no longer
+necessary
+5. Feature's Variables are removed from code
+6. Once it can be verified that the Variables are no longer used in any running code, the Feature on DevCycle can be 
+archived
+
+For an experiment feature:
+1. Feature Variables are added to code and the Feature is set up in DevCycle
+2. Feature is enabled for a subset of users that are part of the experiment
+3. Feature is considered "complete" once it has reached statistical significance, and the winning variation can be determined.
+4. Feature's Variables are removed from code and replaced with the winning variation's values
+5. Once it can be verified that the Variables are no longer used in any running code, the Feature on DevCycle can be
+   archived
+
+## Identifying Flags that should be Removed
+Luckily, DevCycle includes these four feature types as a first-class concept in our platform. Every time you create a 
+new Feature, you are asked to choose which of these four types it falls under. You can then filter your Features by those
+types on the dashboard to keep track of active releases, experiments etc.
+
+DevCycle also includes the ability to mark a feature as "completed", which is a half-way step between being done with
+a feature and fully removing it from your code. Completion allows you to serve a single variation of the feature to all
+users and prevent further changes to its configuration. In this state, DevCycle will ensure your code continues to 
+receive the correct final state of the feature, like the "released" state of a Release-type feature. For more information,
+see [Feature Status & Lifecycle](/essentials/status-and-lifecycle)
+
+The next step is to remove the Feature's Variables from your code, and eventually archive the feature in DevCycle
+so that it no longer appears in the list of active Features.
+
+## Safely Removing Flags
+Once flags that should be removed are identified, the next step is to clean them out of the code. DevCycle provides
+many helpful tools to assist with this.
+
+### Code Usages
+DevCycle's Code Usages feature allows our system to analyze your source code and automatically identify places
+where a given variable is being used. 
+The Code Usages feature can be used locally with our CLI, or as part of an [integration with various source control
```

Comment:
Either add another line break to make this a distinct paragraph or remove the line break

Reply:
not sure what you mean by removing the line break. With or without the single line break it will look the same in the rendered Markdown